geometry_msgs/Vector3[4] vertices
